internal/runners/prepare/prepare_windows.go

```diff
@@ -50,6 +50,12 @@ func (r *Prepare) prepareStartShortcut() error {
 		return locale.WrapError(err, "err_preparestart_icon", "Could not set icon for shortcut file")
 	}
 
+	sc = shortcut.New(shortcutDir, "Uninstall State Tool", filepath.Join(os.Getenv("windir"), "system32", "cmd.exe"), "/C \"state clean uninstall\"")
```

Comment:
The subshell package has a function for getting the shell binary. See [here](https://github.com/ActiveState/cli/blob/master/internal/subshell/subshell.go#L143)

Reply:
Thanks for the tip. Very useful.